src/client/app/model/card.ts

```diff
@@ -1,15 +1,26 @@
-import { Point } from './geometry';
+import { Point, Size } from './geometry';
 import { Arrow } from './arrow';
 import { CollaborativeObjectModel } from './collaborative-object-model';
+import CollaborativeList = gapi.drive.realtime.CollaborativeList;
 
 export class Card extends CollaborativeObjectModel {
   protected static modelName = 'Card';
+  readonly size: Size = {width:120, height:160};
```

Comment:
Indent object literals?

(I'm not a fan of C++ style declaring that vertical whitespace is the devil...)

Reply:
Meh. Alright